includes/group-sites.php

```diff
@@ -1727,11 +1758,16 @@ function cboxol_blogname_is_illegal( $blogname ) {
  * @param array $items Menu items.
  * @return array
  */
-function cboxol_add_links_to_nav_menu( $items ) {
+function cboxol_add_links_to_nav_menu( $items, $args ) {
 	if ( get_current_blog_id() === cboxol_get_main_site_id() ) {
 		return $items;
 	}
 
+	// Skip if we have real nav menu items.
```

Comment:
It looks like `cboxol_custom_menus` is set when a site is initially populated from the template. But if I have an existing site and I manually add a [Group Home] or 'Home' link, should `cboxol_custom_menus()` be set on the menu so that we stop injecting these menu items?

Alternatively, maybe the idea is that we would run a *full* migration routine, where every existing site would have the nav menus added. If we do that, then we probably don't need `cbox_add_links_to_nav_menus()` at all anymore.

Reply:
Resolved via a0c1482.